common/model.go

```diff
@@ -0,0 +1,8 @@
+package common
+
+import (
+	"time"
+)
+
+// HTTPTimeout timeout used in HTTP connections between nodes
+const HTTPTimeout = 8 * time.Second
```

Comment:
Can we add this to `common/util.go` rather than define a new file with just this one constant?

Reply:
cc @ya7ya

Reply:
I've added `model.go` to be place to move interfaces like `Orchestrator` into it (in the future), but if it's too early I can move `HTTPTimeout` into `util.go`

Reply:
Fixed